doc/user-guide/dask.rst

```diff
@@ -2,195 +2,151 @@
 
 .. _dask:
 
-Parallel computing with Dask
+Parallel Computing with Dask
 ============================
 
-Xarray integrates with `Dask <https://dask.org/>`__ to support parallel
-computations and streaming computation on datasets that don't fit into memory.
-Currently, Dask is an entirely optional feature for xarray. However, the
-benefits of using Dask are sufficiently strong that Dask may become a required
-dependency in a future version of xarray.
+`Dask <https://dask.org/?utm_source=xarray-docs>`__, a general purpose library for parallel computing, integrates with Xarray to handle larger-than-memory computations.
```

Comment:
I would stick with saying Xarray integrates with Dask for parallel computation and larger-than-memory computation. This is the Xarray documentation, after all :)

Reply:
Hah, good point! :)